src/h5cpp/datatype/datatype.hpp

```diff
@@ -22,6 +22,7 @@
 // Authors:
```

Comment:
This applies only to **integer** types, so this should actually go with the `integer.hpp` and `integer.cpp` rather than for all datatypes. We want to take advantage of C++ strong typing. There is no sense in answering this for other types. The tests should also go with the the integer type as well.